mailpoet/assets/js/src/form/fields/tokenField.tsx

```diff
@@ -10,12 +19,15 @@ function getItems(endpoint: string) {
   return items;
 }
 
-function FormFieldTokenField(props) {
-  const selectedValues = Array.isArray(props.item[props.field.name])
-    ? props.item[props.field.name].map((item) => props.field.getName(item))
+function FormFieldTokenField(props: TokenFormFieldProps) {
+  const selectedValues: TokenFieldProps['selectedValues'] = Array.isArray(
+    props.item[props.field.name],
+  )
+    ? props.field.name &&
+      props.item[props.field.name].map((item) => props.field.getName(item))
     : [];
 
-  let suggestedValues = [];
+  let suggestedValues;
```

Comment:
thanks for noticing, since we have `else if ` statement the default should stay here, but the reason to touch it has been TS nagging about the `any[]` type assignment, fixed in bacbf08493a1c5f1718e2adebbc8f3e1d83144d0